packages/workspace/src/workspace/state/in_mem.ts

```diff
@@ -148,19 +149,15 @@ export const buildInMemState = (loadData: () => Promise<StateData>, persistent =
     setAll: async (elements: ThenableIterable<Element>): Promise<void> => awu(elements).forEach(setElement),
     remove: removeId,
     isEmpty: async (): Promise<boolean> => (await stateData()).elements.isEmpty(),
-    getAccountsUpdateDates: async () => {
-      const stateDataVal = await awu((await stateData()).accountsUpdateDate.entries()).toArray()
-      return Object.fromEntries(stateDataVal.map(e => [e.key, e.value]))
-    },
-    existingAccounts: async (): Promise<string[]> => awu((await stateData()).accountsUpdateDate.keys()).toArray(),
+    existingAccounts: async (): Promise<string[]> => (await (await stateData()).accounts.get('account_names')) ?? [],
     getPathIndex: async (): Promise<PathIndex> => (await stateData()).pathIndex,
     getTopLevelPathIndex: async (): Promise<PathIndex> => (await stateData()).topLevelPathIndex,
     clear: async () => {
       const currentStateData = await stateData()
       await currentStateData.elements.clear()
       await currentStateData.pathIndex.clear()
       await currentStateData.topLevelPathIndex.clear()
-      await currentStateData.accountsUpdateDate.clear()
+      await currentStateData.accounts.clear()
```

Comment:
we need to keep the existing remote map in the code until we are done with migrating state files (we need to define some time period where we support the migration).
we can't just ignore the fact that the old information exists before we migrate to the new format
```suggestion
      await currentStateData.accountsUpdateDate.clear()
      await currentStateData.accounts.clear()
```

Reply:
Ack